module4/assessments/Week1-3021.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+title: Week 1 Assessment
+---
+
+## Questions and Exercises (10 Points Total)
+
+**Fork** [this repl](https://replit.com/@launch-team/Mod-4-Week-1-Assessment) and complete the questions and exercise it contains.  
+
+
+<aside class="instructor-notes" markdown="1">
+    <p><strong>Instructor Note</strong><br>Here is one solution: https://replit.com/@launch-team/Mod-4-Week-1-Assessment-Solution</p>
+</aside>
+
+### Submission and Feedback
+When completed, click the `Invite` button in the upper right corner and `Generate a join link`.  Include that link in your form submission.
+
+![](/assets/images/module1/Week1/GenerateJoinLink.png)
+
+## Part 1: Questions (10 Points Total)
+
+Answer the questions in the `questions.md` file in the Replit.

```

Comment:
I like the questions:

1. STAR method question is perfect.
2. Debug question is perfect.
3. Have we asked students to perform brand new research during an assessment before?
4. This is all or nothing, I presume. Do they need to get all 5 correct to earn the point?

Reply:
For 3. We have had them google small things in the assessments before, and it's feeling ok to me to lean more into that as we move later in the course and release responsibility.

For 4.  I'm not following what you mean by "get all 5 correct" mean. I have 4 written questions in the assessment and I put the max point values for each in the Repl. I'm imagining we can also do partial points for the multiple-point questions. I'm going to merge this PR in so it's ready for next week, but if you have other concerns or suggestions for changing the grading structure let me know! We can always come back and make an edit.